Tests/test_image_thumbnail.py

```diff
@@ -16,7 +16,7 @@
 
 def test_sanity() -> None:
     im = hopper()
-    assert im.thumbnail((100, 100)) is None
+    assert im.thumbnail((100, 100)) is None  # type: ignore[func-returns-value]
```

Comment:
Without this ignore, mypy reports

> Tests/test_image_thumbnail.py:19: error: "thumbnail" of "Image" does not return a value (it only ever returns None)  [func-returns-value]
>         assert im.thumbnail((100, 100)) is None
>                ^~~~~~~~~~~~~~~~~~~~~~~~

Reply:
Couldn't this line just be `im.thumbnail((100, 100))`, without the assert?

Reply:
Ok, I’ve pushed a commit.